backend/src/validators/projects/deliverables.js

```diff
@@ -0,0 +1,78 @@
+const { Schema, getResponse, getUpdateResponse, getAddResponse } = require("../common_schema");
+const S = require("fluent-json-schema");
+const color = S.object().prop("red", S.number()).prop("green", S.number()).prop("blue", S.number());
```

Comment:
You could add this to common_schema since it'll be used in other places too.

Reply:
future :)